.env

```diff
@@ -0,0 +1 @@
+SKIP_PREFLIGHT_CHECK=true
```

Comment:
- [ ] I would recommend avoiding pushing the `.env` file on Github, Instead, put that in the `.gitignore` file.